Pod/Classes/WPBadgeView.m

```diff
@@ -0,0 +1,141 @@
+
+#import "WPBadgeView.h"
+
+static const CGFloat kDefaultShadowRadius = 2.f;
+static const CGFloat kDefaultShadowOpacity = 0.5f;
+static const CGFloat kDefaultCornerRadius = 6.f;
+static const UIEdgeInsets kDefaultEdgeInsets = {3.f, 6.f, 3.f, 6.f};
+
+@interface WPBadgeView()
+@property (nonatomic, strong) NSLayoutConstraint *topConstraint;
+@property (nonatomic, strong) NSLayoutConstraint *bottomConstraint;
+@property (nonatomic, strong) NSLayoutConstraint* leadingConstraint;
+@property (nonatomic, strong) NSLayoutConstraint* trailingConstraint;
+
+@property (nonatomic, strong) UIView *contentView;
+@property (nonatomic, strong) UIVisualEffectView *blurEffectView;
+@end
+
+@implementation WPBadgeView
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self initialSetup];
+    }
+    return self;
+}
+
+- (instancetype)init
+{
+    self = [super initWithFrame:(CGRectZero)];
+    if (self) {
+        [self initialSetup];
+    }
+    return self;
+}
+
+- (void)initialSetup
```

Comment:
Just to be consistent with other view classes on the pod, can we rename this method to `commonInit` ?